core/io/resource_format_binary.cpp

```diff
@@ -926,8 +918,7 @@ void ResourceInteractiveLoaderBinary::open(FileAccess *p_f) {
 	if (f->eof_reached()) {
 
 		error = ERR_FILE_CORRUPT;
-		ERR_EXPLAIN("Premature End Of File: " + local_path);
-		ERR_FAIL();
+		ERR_FAIL_MSG("Premature End Of File: " + local_path + ".");
```

Comment:
Lowercase "End Of File" (but might be worth adding "EOF", e.g. "Premature end of file (EOF): ".